### PR TITLE
Feature/gm 7669

### PR DESCRIPTION
**Pull request by Tai-DucTran**
Support more customized styles from floating placeholder
Support self-close the popover

### CODE REVIEW

File: test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-chromium-linux.png
Comment (farhanabi) [inline test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-chromium-linux.png:1]: This remove the placeholder?

File: test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-disabled-chromium-linux.png
Comment (farhanabi) [inline test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-disabled-chromium-linux.png:1]: Ditto. This remove the placeholder?

File: test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-chromium-linux.png
Comment (Tai-DucTran) [reply test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-chromium-linux.png:1]: Found the issue @farhanabi 
I removed `{...props}` in <Input>

File: test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-disabled-chromium-linux.png
Comment (Tai-DucTran) [reply test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-disabled-chromium-linux.png:1]: Found the issue @farhanabi 
I removed `{...props}` in <Input>

Comment (farhanabi) [reply test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-disabled-chromium-linux.png:1]: Noice, so it won't change the existing behaviour?

**Review by farhanabi — APPROVED** [review]

**Pull request merged**